airflow/bin/cli.py

```diff
@@ -440,13 +440,23 @@ def serve_logs(filename):  # noqa
             filename,
             mimetype="application/json",
             as_attachment=False)
+
     WORKER_LOG_SERVER_PORT = \
         int(conf.get('celery', 'WORKER_LOG_SERVER_PORT'))
     flask_app.run(
         host='0.0.0.0', port=WORKER_LOG_SERVER_PORT)
 
 
 def worker(args):
+    """
+    Spawn Celery workers for using the CeleryExecutor
+    """
+
+    # only applied when using the CeleryExecutor
+    executor_type = conf.get('core', 'executor')
+    if executor_type != 'CeleryExecutor':
```

Comment:
Sure, added a helper in configuration to get the executor type case insensitive.